test/AutoRest.TestServer.Tests/NonStringEnumTests.cs

```diff
@@ -3,8 +3,8 @@
 
 using System.Threading.Tasks;
 using AutoRest.TestServer.Tests.Infrastructure;
-using non_string_enum;
-using non_string_enum.Models;
+using Non_String_Enum;
```

Comment:
Is this due to a limitation of .NET Core on Linux?

Reply:
We are using Roslyn formatting to make generated files look readable. For some reason, Roslyn sorts `using` directives differently on Linux and Windows in terms of lower case and upper case characters. And it is not configurable yet: https://github.com/dotnet/roslyn/issues/44052
Since we need a fix ASAP, I've come up with this workaround.